### PR TITLE
Feature/eslint

### PR DESCRIPTION
**Pull request by tankakatan**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+        'no-eval': 'error',
+        'no-implied-eval': 'error'
+        // '@typescript-eslint/no-var-requires': 'error', // If we decide to proceed with TS
+        // '@typescript-eslint/ban-ts-comment': 'error', // If we decide to proceed with TS
+        // '@typescript-eslint/no-non-null-assertion': 'off', // If we decide to proceed with TS
+        // '@typescript-eslint/no-shadow': 'error',
+        // '@typescript-eslint/no-unused-vars': 'error'
+        // 'no-shadow': 'off',
+        // 'import/no-extraneous-dependencies': 'error',
+        // 'import/imports-first': ['error', 'absolute-first'],
```
Comment (Koochr) [inline .eslintrc.js:82]: Why did you remove this one? It can be auto-fixed

Comment (tankakatan) [reply .eslintrc.js:82]: I am going to uncomment this as soon as we start using `import` statements.

File: network/web3bridge.js [hunk 2]
```diff
@@ -179,55 +207,63 @@ class Web3Bridge {
...
-                subscriptionId: subscriptionId = id,
-                data: {
-                    message: `Subscribed to "${contractName}" contract "${event}" events with subscription id: ${id}`
-                }
-            }));
+            .on('data', data => onEvent({subscriptionId, data}))
+            .on('connected', id =>
+                onStart({
+                    subscriptionId: (subscriptionId = id),
+                    data: {message: `Subscribed to "${contractName}" contract "${event}" events with subscription id: ${id}`}
```
Comment (tankakatan) [inline network/web3bridge.js:228]: Is this a `prettier` bug? The line length here is 125. So it didn't fix anything, just made the code less readable.

File: scripts/patch-config.js [hunk 3]
```diff
@@ -4,19 +4,24 @@ const {existsSync, writeFileSync, unlinkSync, mkdirSync, readFileSync} = require
...
     Identity: process.env.CONTRACT_ADDRESS_IDENTITY,
     Migrations: process.env.CONTRACT_ADDRESS_MIGRATIONS,
-    StorageProviderRegistry: process.env.CONTRACT_ADDRESS_STORAGE_PROVIDER_REGISTRY,
+    StorageProviderRegistry: process.env.CONTRACT_ADDRESS_STORAGE_PROVIDER_REGISTRY
 };
 
 log.debug(
     'Patching Subprovider rpc: ' +
-    execSync('sed -i \'s/timeout: 20000,/timeout: process.env.SUBPROVIDER_RPC_TIMEOUT || 120000,/g\' /app/node_modules/\\@trufflesuite/web3-provider-engine/subproviders/rpc.js').toString()
+        execSync(
```
Comment (tankakatan) [inline scripts/patch-config.js:22]: Please fix.

File: threads/decrypt.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 #!/bin/env
-
+/* eslint-disable no-console */
```
Comment (tankakatan) [inline threads/decrypt.js:2]: Please remove.

File: threads/encrypt.js [hunk 5]
```diff
@@ -1,5 +1,5 @@
 #!/bin/env
-
+/* eslint-disable no-console */
```
Comment (tankakatan) [inline threads/encrypt.js:2]: Please remove this comment along with `console.log` calls in this file.

File: wallet/index.js [hunk 6]
```diff
@@ -205,21 +244,24 @@ class Wallet {
         return new solana.PublicKey(this.getSolanaAccount());
     }
 
-    async #getSolanaBalanceInSOLWithConnection(connection) {
+    async getSolanaBalanceInSOLWithConnection(connection) {
```
Comment (tankakatan) [inline wallet/index.js:247]: Please restore all the private properties in this file.

**Review by tankakatan — COMMENTED** [review]
@Koochr, please resolve the mentioned issues. Some of them are **errors**, while the others just spoil the code readability.

As it is described in the below comments, using prettier brings serious issues because if forces to use very specific rules that cannot be re-configured. Please see issues like [this](https://github.com/prettier/prettier/issues/2235) and those with array of objects and the way how it formats functions. I strongly recommend not to use it further on.

Some more examples of bad auto-fixes:
```
zweb/deployer/index.js:101
zweb/renderer/index.js:305
models/chunk.js:21
models/file.js:21
models/provider_chunk.js:93
models/redkey.js:18
models/redkey.js:51
kademlia/bson-rpc.js:19
kademlia/index.js:234
update-version.js:16
scripts/identityMigrator.js:4
```

File: console/index.js [hunk 7]
```diff
@@ -1,6 +1,6 @@
 const readline = require('readline');
 const axios = require('axios');
-
+/* eslint-disable no-console */
```
[the remark this answers is not in the document]
Comment (Koochr) [reply console/index.js:3]: This file, same as file below, cannot use logger. So the only option is to use console
